tests/syntax/test_compiler.py

```diff
@@ -7,6 +7,125 @@
 
 
 class TestCompiler:
+    def test_scenic_try(self):
+        node, _ = compileScenicAST(
+            ScenicTry(
+                body=[Expr(Call(Name("foo", Load()), args=[], keywords=[]))],
+                interrupt_when_handlers=[
+                    InterruptWhenHandler(
+                        Constant(True),
+                        [Expr(Call(Name("foo", Load()), args=[], keywords=[]))],
```

Comment:
Let's have the function be "bar" so that we can catch if the body and/or interrupt handlers are put in the wrong place.

Reply:
Thanks for the catch. Fixed in 93a961e4a33b157247cf8364dac7f96d44520097